packages/ember-simple-auth/addon/authenticators/torii.js

```diff
@@ -73,7 +72,7 @@ export default BaseAuthenticator.extend({
       return this.get('torii').fetch(data.provider, data).then(
         (fetchedData) => {
           this._authenticateWithProvider(provider, fetchedData);
-          return emberAssign(data, fetchedData);
+          return Object.assign(data, fetchedData);
```

Comment:
This won't work in IE11 (which I think ESA still supports as long as we support Ember 3.x).

Reply:
Note that ESA already uses Object.assign here:
https://github.com/simplabs/ember-simple-auth/blob/master/packages/ember-simple-auth/addon/session-stores/cookie.js#L183

Reply:
Ouch – thanks for pointing that out!